docs/modules/operation/pages/events/event-configuration.adoc

```diff
@@ -1,39 +1,47 @@
+The overall behavior of Eventd is configured in the file `$\{OPENNMS_HOME}/etc/eventd-configuration.xml`.
 This file does not need to be changed in most installations.
 The configurable items include:
 
 TCPAddress::
-    The IP address to which the `Eventd` XML/TCP listener will bind. Defaults to `127.0.0.1`.
+    The IP address to which the Eventd XML/TCP listener will bind.
+    Defaults to `127.0.0.1`.
 TCPPort::
-    The TCP port number on `TCPAddress` to which the `Eventd` XML/TCP listener will bind. Defaults to `5817`.
+    The TCP port number on `TCPAddress` to which the Eventd XML/TCP listener will bind.
+    Defaults to `5817`.
 UDPAddress::
-    The IP address to which the `Eventd` XML/UDP listener will bind. Defaults to `127.0.0.1`.
+    The IP address to which the Eventd XML/UDP listener will bind.
+    Defaults to `127.0.0.1`.
 UDPPort::
-    The UDP port number on `TCPAddress` to which the `Eventd` XML/UDP listener will bind. Defaults to `5817`.
+    The UDP port number on `TCPAddress` to which the Eventd XML/UDP listener will bind.
+    Defaults to `5817`.
 receivers::
-    The number of threads allocated to service the event intake work done by `Eventd`.
+    The number of threads allocated to service the event intake work done by Eventd.
 queueLength::
-    The maximum number of events that may be queued for processing. Additional events will be dropped. Defaults to unlimited.
+    The maximum number of events that may be queued for processing.
+    Additional events will be dropped.
+    Defaults to unlimited.
 getNextEventID::
-    An SQL query statement used to retrieve the ID of the next new event. Changing this setting is not recommended.
+    An SQL query statement used to retrieve the ID of the next new event.
+    Changing this setting is not recommended.
 socketSoTimeoutRequired::
-    Whether to set a timeout value on the `Eventd` receiver socket.
+    Whether to set a timeout value on the Eventd receiver socket.
 socketSoTimeoutPeriod::
     The socket timeout, in milliseconds, to set if `socketSoTimeoutRequired` is set to `yes`.
 logEventSummaries::
-    Whether to log a simple (terse) summary of every event at level `INFO`. Useful when troubleshooting event processing on busy systems where `DEBUG` logging is not practical.
+    Whether to log a simple (terse) summary of every event at level `INFO`.
+    Useful when troubleshooting event processing on busy systems where `DEBUG` logging is not practical.
 
 
 The set of known events is configured in `$\{OPENNMS_HOME}/etc/eventconf.xml`.
-This file opens with a `<global>` element, whose `<security>` child element defines which event fields may not be overridden in the body of an event submitted via any `Eventd` listener.
+This file opens with a `<global>` element, whose `<security>` child element defines which event fields may not be overridden in the body of an event submitted via any Eventd listener.
```

Comment:
The action being done is reading from the file, which would be an "override" as opposed to saving change and writing over an existing value.  But it could probably go either way.  I don't have strong feelings on this and will go with whichever you think is best here.

Reply:
Thanks, I wasn't sure if there was a distinction, so let's keep override.

Reply:
I dunno.  That's just based on the logic in my head at the moment. 😬